server/events/markdown_renderer.go

```diff
@@ -73,6 +74,11 @@ type planSuccessData struct {
 	PlanWasDeleted bool
 }
 
+type policyCheckSuccessData struct {
+	models.PolicyCheckSuccess
+	PlanWasDeleted bool
```

Comment:
Not sure if this is needed, policies are not ran if plans are being deleted. It is an explicit condition in the command_runner.

Reply:
What if we lose the ebs volume and the plan file gets deleted?

Reply:
I guess I can remove it for now